src/main/scala/io/iohk/ethereum/nodebuilder/StdNode.scala

```diff
@@ -132,6 +134,24 @@ abstract class BaseNode extends Node {
     tryAndLogFailure(() => Metrics.get().close())
     tryAndLogFailure(() => storagesInstance.dataSource.close())
   }
+
+  def fixDatabase(): Unit = {
```

Comment:
I added this to be able to maintain compatibility.
The problem is that when we start with an existing database (say staging), we don't have the best block hash stored. This ensures on startup that the database is consistent. 
@LeonorLunatech, @strauss-m, do you think it is an acceptable compromise ?

Reply:
👍 Let as know how the tests went

Reply:
It worked as expected and staging is mining normally 👍